### ISSUE TITLE
Improve visibility of focus indicator

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#93]
visible focus indicator — In Chrome, the visible focus indicator is Hex #66AFE9 which fails contrast requirements. Consider changing the focus indicator to a thicker blue line, Hex #477BA5, with a halo so that it is consistent across browsers and more perceivable.
